docs/setup_installation/azure/getting_started.md

```diff
@@ -1,373 +1,193 @@
+- Create a managed identity
+- Create a role with appropriate permission and assign it to the managed identity
 
-<p align="center">
-  <figure>
-    <img style="border: 1px solid #000;width:700px" src="../../../assets/images/setup_installation/managed/azure/create-acr.png" alt="Create a Container Registry">
-    <figcaption>Create a Container Registery</figcaption>
-  </figure>
-</p>
+### Step 4.1: Create a managed identity
+You create a managed identity by running the following command, replacing *$RESOURCE_GROUP* with the name of your resource group.
 
-### Step 3.2: Give the Managed Identity access to the container registry
+```bash
+identityId=$(az identity create --name hopsworks-instance --resource-group $RESOURCE_GROUP --query principalId -o tsv)
+```
 
-Go to the managed identity created in [Step 2.2](#step-22-creating-a-user-assigned-managed-identity). Click on *Azure role assignments* in the left column. Click on *Add role assignment*. For the *Scope* select *Resource group* or *Subscription* depending on your preference. Select the *Role* *AcrPull* and click on *Save*.
+### Step 4.2: Create a role for the managed identity
+To create a new role for the managed identity, first, create a file called *instance-role.json* with the following content. Replace *SUBSCRIPTION_ID* by your subscription id and *RESOURCE_GROUP* by your resource group
 
-Repeat the same operation with the following roles:
+```json
+{
+  "Name": "hopsworks-instance",
+  "IsCustom": true,
+  "Description": "Allow the hopsworks instance to access the storage and the docker repository",
+  "Actions": [
+      "Microsoft.Storage/storageAccounts/blobServices/containers/write",
+      "Microsoft.Storage/storageAccounts/blobServices/containers/read",
+      "Microsoft.Storage/storageAccounts/blobServices/write",
+      "Microsoft.Storage/storageAccounts/blobServices/read",
+      "Microsoft.Storage/storageAccounts/listKeys/action",
+      "Microsoft.ContainerRegistry/registries/artifacts/delete",
+      "Microsoft.ContainerRegistry/registries/pull/read",
+      "Microsoft.ContainerRegistry/registries/push/write"
+  ],
+  "NotActions": [
+
+  ],
+  "DataActions": [
+      "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/delete",
+      "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read",
+      "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/move/action",
+      "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/write"
+  ],
+  "AssignableScopes": [
+    "/subscriptions/SUBSCRIPTION_ID/resourceGroups/RESOURCE_GROUP"
+  ]
+}
+```
+Then run the following command, replacing *$RESOURCE_GROUP* with the name of your resource group, to create the new role.
```

Comment:
it seems role definition creation doesn't require resource group to be set on command line
```suggestion
Then run the following command, to create the new role.
```